packages/vx-axis/src/axis/Axis.js

```diff
@@ -58,72 +68,66 @@ export default function Axis({
       y: horizontal ? 0 : range1,
     });
 
+    const labelFontSize = labelComponent.props.fontSize || 10;
+    const tickLabelFontSize = tickLabelComponent.props.fontSize || 10;
+
+    const labelProps = {
+      transform: horizontal ?  '' : `rotate(${tickSign * 90})`,
+      x: horizontal ? range1 / 2 : tickSign * (range0 / 2),
+      y: horizontal ?
+        (tickSign * (tickLength + tickLabelOffset + tickLabelFontSize + (!isTop ? labelFontSize : 0)))
+        : (isLeft ? 1 : -1) * tickSign * (tickLength + tickLabelOffset),
+    };
```

Comment:
i'm not opposed to keeping this as is, but it'd be great to turn this into a `getTransfrom()` function passing in the needed props... maybe then we could pass in a `labelTransform()` prop that let folks override it with their own thing if they wanted... not sold on that idea, just throwing that out there.